src/PythonProjectBootstrapper/ProjectGenerationUtils.py

```diff
@@ -79,31 +82,52 @@ def ConditionallyRemoveUnchangedTemplateFiles(
             original_hash = existing_manifest_dict[removed_file_rel_path]
 
             if current_hash == original_hash:
+                deleted_files.add(removed_full_path.as_posix())
                 removed_full_path.unlink()
 
+    return deleted_files
+
 
 # ----------------------------------------------------------------------
 def CopyToOutputDir(
     src_dir: Path,
     dest_dir: Path,
-) -> None:
+) -> list[set[str]]:
     # Copies all generated files into the output directory and handles the creation/updating of the manifest file
 
     PathEx.EnsureDir(src_dir)
     PathEx.EnsureDir(dest_dir)
 
+    prompt_file = src_dir / "prompt_text.yml"
+    if prompt_file.is_file():
+        shutil.copy2(prompt_file, dest_dir)
+        prompt_file.unlink()
+        PathEx.EnsureFile(dest_dir / "prompt_text.yml")
+
     # existing_manifest will be populated/updated as necessary and saved
     generated_manifest: dict[str, str] = CreateManifest(src_dir)
     existing_manifest: dict[str, str] = {}
 
+    overwritten_files: set[str] = set()
```

Comment:
Consider changing these to `list` types rather than `set`. Lists preserve order, while sets do not. I imagine that these values are being displayed somewhere, and it might be a bit jarring for a user to see them in what appears to be random order.